app/scripts/auto-type/auto-type-parser.js

```diff
@@ -1,7 +1,8 @@
 import { AutoTypeRunner } from 'auto-type/auto-type-runner';
 
-const AutoTypeParser = function(sequence) {
+const AutoTypeParser = function(sequence, windowID) {
```

Comment:
The parser doesn't need this data, let's remove it from here and pass to the runner in its `run` method